scipy/optimize/optimize.py

```diff
@@ -2387,7 +2387,14 @@ def fmin_powell(func, x0, args=(), xtol=1e-4, ftol=1e-4, maxiter=None,
         iteration.  Called as ``callback(xk)``, where ``xk`` is the
         current parameter vector.
     direc : ndarray, optional
-        Initial direction set.
+        Initial fitting step and parameter order set as an NxN vector, where N
+        is the number of fitting parameters in x0. Defaults to step size 1.0
+        fitting all parameters simultaneously ``[[1]*N]*N``. To prevent initial
```

Comment:
If you mean all ones matrix, you can use `ones((N, N))` here. NumPy requirement is implicit in the documentation.